.enb/config/libs.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+    'bem-core' : {
+        path : 'libs/bem-core',
+        levels : {
+            desktop : ['libs/bem-core/common.blocks', 'libs/bem-core/desktop.blocks'],
+            touch : ['libs/bem-core/common.blocks', 'libs/bem-core/touch.blocks'],
+            'touch-pad' : ['libs/bem-core/common.blocks', 'libs/bem-core/touch.blocks'],
+            'touch-phone' : ['libs/bem-core/common.blocks', 'libs/bem-core/touch.blocks']
+        }
+    },
+    'bem-pr' : {
+        path : 'libs/bem-pr',
+        levels : {
+            spec : ['libs/bem-pr/spec.blocks']
+        }
+    },
+    'design' : {
```

Comment:
Its behavior is no different from the usual library. The only difference is that it is inside `bem-components`.